README.md

```diff
@@ -52,6 +52,12 @@ the lints package:
 include: package:lints/core.yaml
 ```
 
+or:
```

Comment:
Depending on the decision in https://github.com/dart-lang/lints/issues/20, we should invert this.  Or maybe *just* suggest the recommended set?

Reply:
Makes sense - I updated the PR two switch core and recommended here.